README.md

```diff
@@ -112,6 +113,35 @@ Default path to cache directory: `node_modules/.cache/uglifyjs-webpack-plugin`.
 
 Path to cache directory.
 
+### `additionalCacheKeys`
+
+#### `{Object}`
+
+**webpack.config.js**
+```js
+[
+  new UglifyJsPlugin({
+    cache: true,
+    additionalCacheKeys: {
+      env: 'development',
```

Comment:
Bad example, many people can use this from `README`, just use `myCustomVariable` name or something like